sockeye/init_embedding.py

```diff
@@ -133,11 +134,11 @@ def main():
     args = params.parse_args()
 
     if len(args.weight_files) != len(args.vocabularies_in) or \
-       len(args.weight_files) != len(args.vocabularies_out) or \
-       len(args.weight_files) != len(args.names):
-           logger.error("Exactly the same number of 'input weight files', 'input vocabularies', "
-                        "'output vocabularies' and 'Sockeye parameter names' should be provided.")
-           sys.exit(1)
+            len(args.weight_files) != len(args.vocabularies_out) or \
+            len(args.weight_files) != len(args.names):
+        logger.error("Exactly the same number of 'input weight files', 'input vocabularies', "
+                     "'output vocabularies' and 'Sockeye parameter names' should be provided.")
+        sys.exit(1)
```

Comment:
It takes a minute to see that this doesn't change the logic.  Good to have consistent use of whitespace!